package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "the-wall-templater-dustjs",
   "description": "The Wall compatible template handler",
-  "version": "3.1.1",
+  "version": "3.1.7",
```

Comment:
yes, there were a few very minor changes that I bumped the version for, in hindsight I wouldn't have done